front/app/containers/ProjectsShowPage/timeline/setPhaseURL.ts

```diff
@@ -19,5 +19,7 @@ export default function setPhaseUrl(
       ? projectURL
       : `${projectURL}/${phaseNumber}`;
 
-  window.history.pushState(null, '', url);
+  const search = window.location.search;
+
+  window.history.pushState(null, '', url + search);
```

Comment:
Why use the window history here? Let's use react router somehow, if possible?

Reply:
I'll use `clHistory` instead for pushing to the history, but I think using `window.location.search` is okay here, since I need it as one big string and not as a `URLSearchParams` interface (and that also requires me to set up a hook call in whichever component uses this)